### ISSUE TITLE
time

### ISSUE THREAD

**Issue by GoodGoodStudy-AI** [matterport/Mask_RCNN#1658]
I'm running train_shapes.ipynb in a anaconda environment. 
My GPU is Titan RTX
Ubuntu18.04
cuda10.0
cdunn7.6.0
Tensorflow-gpu tf.__version__==1.3.0
keras 2.0.8
Mask R-CNN 2.1  https://github.com/matterport/Mask_RCNN/releases

As described in train_shapes.ipynb:  'On a GPU, you can start to get okay-ish results in a few minutes, and good results in less than an hour'. 
And the original output shows: 
Epoch 1/1
100/100 [==============================] - 73s - loss: 2.2164 - rpn_class_loss: 0.0242 - rpn_bbox_loss: 1.0638 - mrcnn_class_loss: 0.2426 - mrcnn_bbox_loss: 0.3006 - mrcnn_mask_loss: 0.2385 - val_loss: 1.8454 - val_rpn_class_loss: 0.0232 - val_rpn_bbox_loss: 0.9971 - val_mrcnn_class_loss: 0.1398 - val_mrcnn_bbox_loss: 0.1343 - val_mrcnn_mask_loss: 0.2042

But on my machine it takes more than an hour to run this file with defaulted parameters.
Dose this performance relates to the poor FP64 performance of Titan RTX or some reason else?
